shared/locales/fr.js

```diff
@@ -46,9 +46,9 @@ module.exports = {
 
         charts: {
             Amount: 'Montant',
-            balance: 'balance',
-            by_category: 'par catégorie',
-            differences_all: 'rentrées et sorties (tous les comptes)',
+            balance: 'Solde',
+            by_category: 'Par catégorie',
+            differences_all: 'Rentrées et sorties (tous les comptes)',
```

Comment:
I like the fact that it isn't capitalized, but that might be just me. However, if you change capitalization, can you also change it in the `en` locale?